### DISCUSSION TITLE
How to implement a function similar to ace jump in emacs?

### DISCUSSION THREAD

**Discussion by xulis** [helix-editor/helix#4115] [Q&A]
How to implement a function similar to ace jump in emacs?

I think ace jump is the most efficient way to move the cursor. Can the helix editor implement this function?

Ref: https://plugins.jetbrains.com/plugin/7086-acejump

**Comment by the-mikedavis** [comment]
A jump mode is being worked on in https://github.com/helix-editor/helix/pull/3791

**Reply by xulis** [reply]
very much looking forward to 👍
